pop/POPAnimation.mm

```diff
@@ -270,3 +270,43 @@ - (id)pop_animationForKey:(NSString *)key
 }
 
 @end
+
+@implementation POPAnimation (NSCopying)
+
+- (instancetype)copyWithZone:(NSZone *)zone
+{
+  POPAnimation *copy = nil;
+  
+  // Must use [self class] instead of POPAnimation so that subclasses can call this via super.
+  // Since POPAnimation and POPPropertyAnimation are (semi-)concrete, wrap initialization in a try-catch.
+  // If regular initialization fails, use _init.
```

Comment:
Hm, not sure I fully understand this. How could you copy a POPAnimation if you couldn't init one to start with?